src/main/webapp/resources/js/pages/announcement/components/AnnouncementDashboard.jsx

```diff
@@ -16,19 +16,19 @@ import { PriorityFlag } from "./PriorityFlag";
 
 export function AnnouncementDashboard() {
   const [unreadAnnouncements, setUnreadAnnouncements] = useState([]);
-  const [unreadTotal, setUnreadTotal] = useState(0);
+  const [readToggle, setReadToggle] = useState(false);
```

Comment:
Kind of confused as to what this is doing.  Would it not just be easier in your `markAnnoucementAsRead` function, on a success response to just remove the announcement from the array then to fetch all over again (although I suppose there could be a time when a new announcement is added while the one is being deleted).

Reply:
I can go either way! I chose to call the API just in case there was something new. I can just update the unreadAnnouncements list instead. Do you prefer that?

Reply:
Please see d0d931f41ff1441bc33c61938992fd1ff62310ad.